src/Nox.Generator/Infrastructure/Persistence/EntityDtoSqlQueryBuilderGenerator.cs

```diff
@@ -11,18 +14,18 @@ internal class EntityDtoSqlQueryBuilderGenerator : INoxCodeGenerator
     public NoxGeneratorKind GeneratorKind => NoxGeneratorKind.Infrastructure;
 
     public void Generate(
-      SourceProductionContext context,
-      NoxCodeGenConventions noxCodeGenCodeConventions,
-      GeneratorConfig config,
-      System.Action<string> log,
-      string? projectRootPath)
+        SourceProductionContext context,
+        NoxCodeGenConventions noxCodeGenCodeConventions,
+        GeneratorConfig config,
+        System.Action<string> log,
+        string? projectRootPath)
     {
         context.CancellationToken.ThrowIfCancellationRequested();
 
         if (noxCodeGenCodeConventions.Solution.Domain is null)
             return;
 
-        foreach (var entity in noxCodeGenCodeConventions.Solution.Domain.Entities.Where(e => e.IsLocalized))
+        foreach (var entity in noxCodeGenCodeConventions.Solution.Domain.Entities.Where(e => e.IsLocalized || e.Attributes.Any(x => x.Type == NoxType.Enumeration)))
```

Comment:
Here we should check IsLocalizedEnum, enum may not be sufficient?

For clarity them should be IsLocalizedText and not IsLocalized?